indigo/bulk_creator.py

```diff
@@ -1223,7 +1223,7 @@ def find_work(self, given_string):
             # but it could be in the current batch
             for row in self.works:
                 if hasattr(row, 'work') and (substring == row.work.frbr_uri or given_string == row.work.title):
-                    work = _('%s (about to be imported)') % given_string
```

Comment:
this has to remain as-is, because the interpolation has to happen after the translation (the `_(...)`).